File: internal/collector/treesitter/testdata/test_kind/cpp/cpp-gtest/helper/helpers_test.cc
// SPDX-License-Identifier: Apache-2.0

#include <string>

// Helper functions in a *_test.cc file — classifyTestKindCpp returns
// TestKindHelper for any declaration in a recognized test file.
std::string make_user(const std::string& name) {
  return "user-" + name;
}

File: internal/collector/treesitter/testdata/test_kind/java/java-junit4/helper/HelpersTest.java
// SPDX-License-Identifier: Apache-2.0

package fixture;

// In a *Test.java file, methods without recognized annotations classify as
// TestKindHelper per classifyTestKindJava (chunker_java.go:47). The class
// itself is also TestKindHelper per the class-declaration arm.
public class HelpersTest {
    public String fixtureFor(String name) {
        return "fixture-" + name;
    }
}
